File: Week-04/day-01/sharpie.py
class Sharpie(object):
    def __init__(self, color, width):
        self.color = color
        self.width = float(width)
        self.ink_amount = 100

    def use(self):
        self.ink_amount -= 1

sharpie = Sharpie("green", 3.2)
print(sharpie.color, sharpie.width, sharpie.ink_amount)
sharpie.use()
print(sharpie.color, sharpie.width, sharpie.ink_amount)

File: Week-08/day-01/sharpie.js
class Sharpie {

    constructor(color, width) {
        this.color = color;
        this.width = width;
        this.ink = 100;
    }

    use() {
        this.ink -= this.width;
    }
}

var first_sharpie = new Sharpie("purple", 15);
var second_sharpie = new Sharpie("pink", 1);
first_sharpie.use();
first_sharpie.use();
console.log(first_sharpie.width);
console.log(first_sharpie.ink);

console.log(second_sharpie.ink);
while (second_sharpie.ink > 0) {
  second_sharpie.use();
  console.log(second_sharpie.ink);
}